addons/portal/static/src/chatter/portal/thread_model_patch.js

```diff
@@ -0,0 +1,17 @@
+import { Thread } from "@mail/core/common/thread_model";
+
+import { patch } from "@web/core/utils/patch";
+
+const threadPatch = {
+    getFetchParams() {
+        const fetchParams = super.getFetchParams();
+        if (this.model === "discuss.channel" || this.model === "mail.box") {
+            return fetchParams;
+        }
```

Comment:
channel concept should only be present in `discuss` folder, this should be overridden there

Reply:
But this is not the best way of doing it, as it relies on bundles rather than code, which is something we want to avoid.
There should be a flag/param, coming from whoever is calling this method, that determines the context of the request (a chatter in the portal or something else in this case).

Reply:
So in other words the fetch_portal_message flag does not depend on the thread, but on the context where it is displayed. With the rename suggestion (from_portal/only_portal) it might become more apparent.